pkg/sqlutil/sqltest/sqltest.go

```diff
@@ -86,7 +87,19 @@ func CreateOrReplace(t testing.TB, u url.URL, dbName string, template string) ur
 		t.Fatalf("unable to create postgres test database with name '%s': %v", dbName, err)
 	}
 	u.Path = fmt.Sprintf("/%s", dbName)
-	t.Cleanup(func() { assert.NoError(t, drop(u)) })
+	// simple best effort; some tests seem to hold a db connection and race with this drop
+	t.Cleanup(func() {
+		var err error
+		attempts := 10
+		for i := 0; i < attempts; i++ {
```

Comment:
nit/ You could do this now :shrug:
```suggestion
		for range 10 {
```